docs/components/radio_buttons.md

```diff
@@ -0,0 +1,369 @@
+-->
+
+# Selection controls: radio buttons
+
+[Selection controls](https://material.io/components/selection-controls#usage) allow the user to select options.
+
+Use radio buttons to:
+
+* Select a single option from a list
+* Expose all available options
+* If available options can be collapsed, consider using a dropdown menu instead, as it uses less space.
+
+![Radio button hero example for menu options](assets/RadioButton-hero.png)
+
+## Contents
+
+* [Using radio buttons](#using-radio-buttons)
+* [Import radio buttons](#import-radio-buttons)
+* [Making radio buttons accessible](#making-radio-buttons-accessible)
+* [Radio buttons example](#radio-buttons-example)
+* [Theming](#theming)
+
+## Using radio buttons
+
+Radio buttons allow the user to select one option from a set. Use radio buttons when the user needs to see all available options. If available options can be collapsed, consider using a dropdown menu because it uses less space.
+
+### Import radio buttons
+
+Before you can use radio buttons, you need to import the Material Components package for Flutter:
+
+```dart
+package:flutter/material.dart.
+```
+
+You need to be using a [`MaterialApp`](https://api.flutter.dev/flutter/material/MaterialApp-class.html).
+
+For more information on getting started with the Material for Flutter, go to the Flutter [Material library](https://api.flutter.dev/flutter/material/material-library.html) page.
+
+### Making radio buttons accessible
+
+Flutter's APIs support accessibility setting for large fonts, screen readers, and sufficient contrast. For more information, go to Flutter's [accessibility](https://flutter.dev/docs/development/accessibility-and-localization/accessibility) and [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) pages.
+
+For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
+
+## Radio buttons
+
+### Radio buttons example
+
+'Radio'
+* [Class definition](https://api.flutter.dev/flutter/material/Radio-class.html)
+* [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/radio.dart)
+* [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/radio_buttons/regular)
+
+The following example shows a regular FAB.
+
+!["Regular radio button example for Flutter."](assets/radio_buttons/radio-button-regular.png)
+
+```dart
+Column(
+  children: <Widget>[
+    for (int i = 1; i <= 5; i++)
+      ListTile(
+        title: Text(
+          'Radio $i',
+          style: Theme.of(context).textTheme.subtitle1.copyWith(color: i == 5 ? Colors.black38 : Colors.black),
+        ),
+        leading: Radio(
+          value: i,
+          groupValue: _value,
+          activeColor: Color(0xFF6200EE),
+          onChanged: i == 5 ? null : (int value) {
```

Comment:
Should the comments above mention that the 5th radio button is disabled?

Reply:
good point, updated